.github/workflows/gpuci.yml

```diff
@@ -0,0 +1,41 @@
+name: "GPU CI/CD"
```

Comment:
Used [NeMo's cicd-main.yml](https://github.com/NVIDIA/NeMo/blob/main/.github/workflows/cicd-main.yml) as a reference for creating this.